packages/vscode/package.json

```diff
@@ -128,16 +123,6 @@
         "command": "prisma.restartLanguageServer",
         "title": "Restart Language Server",
         "category": "Prisma"
-      },
-      {
-        "command": "prisma.filewatcherEnable",
-        "title": "Enable the File Watcher functionality for Prisma Client.",
-        "category": "Prisma"
-      },
-      {
-        "command": "prisma.filewatcherDisable",
-        "title": "Disable the File Watcher functionality for Prisma Client.",
-        "category": "Prisma"
```

Comment:
Do we really need no configuration about this at all any more?

Reply:
I could restore it if you think it's necessary. I recall we added it because of performance issues, so I thought removing it would make sense.

Reply:
I don't know. Not sure if there are cases where you would not want to trigger this, or be able to completely turn off the file watcher logic.

Reply:
I'll restore it, just in case something goes wrong for a user.

Reply:
Restored.